src/Provider/Google.php

```diff
@@ -121,31 +122,62 @@ protected function createResourceOwner(array $response, AccessToken $token)
     {
         $user = new GoogleUser($response);
 
-        $this->assertMatchingDomain($user->getHostedDomain());
+        $this->assertMatchingDomains($user->getHostedDomain());
 
         return $user;
     }
 
+    protected static function isDomainExpression($str) {
```

Comment:
PSR-2 requires that the opening brace is on a new line.

And I don't understand why this method is static...